assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -656,3 +656,9 @@ che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
 
 # Docker image for the Che async storage
 che.infra.kubernetes.async.storage.image=quay.io/eclipse/che-workspace-data-sync-storage:latest
+
+# The timeout for shutdown Asynchronous Storage Pod after stopping last used workspace.
+che.infra.kubernetes.async.storage.shutdown_timeout_min=120
```

Comment:
Since the async pod runs as a `pod` and not `deployment`, can automatic OpenShift functionality interfere here? IIRC we had to switch to using deployments for workspaces ages ago because OpenShift can remove long-running pods automatically.

Reply:
I created https://github.com/eclipse/che/issues/17616 to discuss this more

Reply:
Yes, this good point, I also think about that